documentation/tutorials/get-started.md

```diff
@@ -530,55 +546,76 @@ update :assign do
-  # If there is a different representative for this ticket, it will be changed to the new one
-  # The representative itself is not modified in any way
+  # We use a change here to replace the related Representative
+  # If there is a different representative for this Ticket, it will be changed to the new one
+  # The Representative itself is not modified in any way
   change manage_relationship(:representative_id, :representative, type: :append_and_remove)
 end
 ```
 
-Lets try it out!
+Let's try it out in our `iex` console!
+
+Use `recompile` to pick up changes you've made to your code, or just restart your session.
 
 ```elixir
-# Use this to pick up changes you've made to your code, or restart your session
 recompile()
+```
 
-# Open a ticket
+### Open a Ticket
+
+```elixir
 ticket = (
   Helpdesk.Support.Ticket
   |> Ash.Changeset.for_create(:open, %{subject: "I can't find my hand!"})
   |> Helpdesk.Support.create!()
 )
+```
+
+### Create a Representative
 
-# Create a representative
+```elixir
 representative = (
   Helpdesk.Support.Representative
   |> Ash.Changeset.for_create(:create, %{name: "Joe Armstrong"})
   |> Helpdesk.Support.create!()
 )
+```
 
-# Assign that representative
+### Assign that Representative to the Ticket
+
+```elixir
 ticket = (
   ticket
   |> Ash.Changeset.for_update(:assign, %{representative_id: representative.id})
   |> Helpdesk.Support.update!()
 )
 ```
 
+> Do we use the last assigned ticket?
+> When we say the "The representative itself is not modified in any way" isn't it updating both sides of the relationship?
+
 ### What next?
 
-What you've seen above constitutes some very simple usage of Ash, barely scratching the surface. In a lot of ways, it will look very similar to other tools that you've seen. If all that you ever used was the above, then realistically you won't see much benefit to using Ash over other options. Where Ash shines is in all of the additional turn-key tools that are built in, the ability to extend the framework yourself, and the consistent design patterns that enable unparalleled velocity, power and flexibility as your application and needs grow.
+What you've seen above barely scratches the surface of what Ash can do. In a lot of ways, it will look very similar to other tools that you've seen. If all that you ever used was the above, then realistically you won't see much benefit to using Ash over other options like Ecto.
+
+> Long awkward sentence
+
+Where Ash shines is in all of the tools that can operate on your resources, the ability to extend the framework yourself, and the consistent design patterns that enable unparalleled efficiency, power and flexibility as your application and needs grow.
 
 #### Clean up your code that uses Ash?
 
 Creating and using changesets can be verbose. Check out the {{link:ash:guide:Code Interface}} to derive things like `Helpdesk.Support.Ticket.assign!(representative.id)`
 
+> Not sure what this bit is about, needs expanding
```

Comment:
Isn't the link to the other guide the expanding?

Reply:
Sort of, a short high level description would be useful here